src/components/sbb-dialog/sbb-dialog.scss

```diff
@@ -13,37 +13,22 @@
   --sbb-dialog-max-width: 100%;
   --sbb-dialog-max-width-default: #{sbb.px-to-rem-build(892)};
   --sbb-dialog-max-height: 100%;
+  --sbb-dialog-inset: 0 auto auto 0;
   --sbb-dialog-border-radius: var(--sbb-border-radius-8x);
   --sbb-dialog-padding-block: var(--sbb-spacing-responsive-s);
   --sbb-dialog-animation-duration: var(--sbb-animation-duration-6x);
   --sbb-dialog-animation-easing: ease;
+  --sbb-dialog-pointer-events: none;
+  --sbb-dialog-backdrop-visibility: hidden;
+  --sbb-dialog-backdrop-pointer-events: none;
   --sbb-dialog-backdrop-color: transparent;
   --sbb-dialog-header-padding-block: var(--sbb-spacing-responsive-s) 0;
   --sbb-dialog-footer-border: var(--sbb-border-width-1x) solid var(--sbb-color-cloud-default);
   --sbb-scrollbar-color: var(--sbb-color-black-alpha-30);
   --sbb-scrollbar-color-hover: var(--sbb-color-black-alpha-60);
 
-  position: fixed;
-  pointer-events: none;
-  display: flex;
-  align-items: center;
-  inset: 0;
-  z-index: 10;
-
-  // Backdrop
-  &::before {
-    content: '';
-    visibility: hidden;
-    pointer-events: none;
-    position: fixed;
-    inset: 0;
-    background-color: var(--sbb-dialog-backdrop-color);
-    transition: {
-      duration: var(--sbb-dialog-animation-duration);
-      timing-function: var(--sbb-dialog-animation-easing);
-      property: background-color, visibility;
-    }
-  }
+  position: absolute;
```

Comment:
does this really work in case e.g. some outer element in the DOM has position: relative on it?
Maybe it helps if we just don't apply any display property to the host, so it has also no space in view. This comment would affects also all other dialog components.